plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -363,7 +371,10 @@ export function wrapFieldResolver<TSource = any, TContext = any, TArgs = any>(
 
     let field: any;
     let shouldEndSpan = false;
-    if (config.depth >= 0 && config.depth < depth) {
+    if (
+      (config.depth >= 0 && config.depth < depth) ||
+      ignoreMap().has(path[path.length - 1])
```

Comment:
Can we use `info.fieldName` instead of `path[path.length - 1]`?
Seems to me more straightforward and robust, than using the output of the `pathToArray`